pkg/engine/engine.go

```diff
@@ -561,46 +567,26 @@ func Start(info *Info) error {
 		return err
 	}
 
-	dir, err := os.Getwd()
-	if err != nil {
-		return err
-	}
-	for instanceID, include := range busConfig.Includes {
-		if oci.IsImageReference(include.Ref) {
-			return errors.New("references not currently supported")
-		}
-		log.Info("Loading include", "name", instanceID, "ref", include.Ref)
-		path := filepath.Join(dir, include.Ref)
-		var busFile, parentDir string
-		info, err := os.Stat(path)
+	for name, include := range busConfig.Includes {
```

Comment:
I think this was my mistake but this construct is supposed to be called "uses" (or maybe import). I see "includes" as more of a file inclusion for breaking up the YAML into multiple files. Not as a Iota boundary.